ivy/functional/frontends/paddle/tensor/stat.py

```diff
@@ -9,3 +9,5 @@
 @to_ivy_arrays_and_back
 def mean(input, axis=None, keepdims=False, out=None):
     return ivy.std(input, axis=axis, keepdims=keepdims, out=None)
+
```

Comment:
Extra new line here isn't needed. pre-commit should help auto fix these style issues - https://unify.ai/docs/ivy/overview/contributing/setting_up.html#pre-commit